README.md

```diff
@@ -97,6 +97,9 @@ The `pm-install` generator generates commands for several JavaScript package man
     <!-- AUTOMD_START generator="pm-install" name="package-name" dev="true" -->
 
     ```sh
+    # ✨ Auto-detect
+    npx nypm@latest add -D package-name
+
```

Comment:
No shade towards `nypm` at all ahah but I don't see many people using it / including it in READMEs. Maybe make it opt-in?

Reply:
Indeed. But well I made this tool mainly for unjs repos and this can actually help this. And indeed it is not widely used because i never pushed for it for end-users (many are already indirectly using nypm using nuxt cli, giget cli, etc)

Reply:
Fair!